src/Models/TraceabilityLink.php

```
@@ -8,36 +8,130 @@
+                    ':from_type' => $fromType,
+                    ':from_id' => $fromId,
+                    ':to_type' => $toType,
+                    ':to_id' => $toId,
+                    ':relationship' => (string) $data['relationship'],
+                    ':confidence' => $data['confidence'] ?? null,
+                    ':metadata_json' => self::jsonOrNull($data['metadata_json'] ?? null),
+                ]
+            );
+
+            if ($stmt->rowCount() === 0) {
+                throw new \RuntimeException('Project does not belong to organisation');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '75', column '27'). (undefined)

(MissingImport)